zk/legacy_executor_verifier/legacy_executor_verifier.go

```diff
@@ -31,90 +29,76 @@ var ErrNoExecutorAvailable = fmt.Errorf("no executor available")
 
 type VerifierRequest struct {
 	BatchNumber  uint64
+	BlockNumbers []uint64
 	ForkId       uint64
 	StateRoot    common.Hash
 	Counters     map[string]int
 	creationTime time.Time
 }
 
-func NewVerifierRequest(batchNumber, forkId uint64, stateRoot common.Hash, counters map[string]int) *VerifierRequest {
+func NewVerifierRequest(forkId, batchNumber uint64, blockNumbers []uint64, stateRoot common.Hash, counters map[string]int) *VerifierRequest {
 	return &VerifierRequest{
 		BatchNumber:  batchNumber,
+		BlockNumbers: blockNumbers,
 		ForkId:       forkId,
 		StateRoot:    stateRoot,
 		Counters:     counters,
 		creationTime: time.Now(),
 	}
 }
 
-func (vr *VerifierRequest) isOverdue() bool {
+func (vr *VerifierRequest) IsOverdue() bool {
 	return time.Since(vr.creationTime) > time.Duration(30*time.Minute)
```

Comment:
this should be a setting

Reply:
You mean the time interval of 30 minutes?

Reply:
I've added a flag

Reply:
Yes, perfect, yes 30 minutes.